src/Uno.Extensions.Templates/content/unoapp-extensions/build/steps/install-dependencies.yml

```diff
@@ -0,0 +1,123 @@
+parameters:
```

Comment:
Please rework this to use uno-check <-- this should be our standard tool for installing dependencies and is used across nearly all existing builds